metricbeat-tests/apache_test.go

```diff
@@ -1,35 +0,0 @@
-package main
-
-import (
-	"github.com/DATA-DOG/godog"
-	"github.com/elastic/metricbeat-tests-poc/cli/services"
-)
-
-var apacheService services.Service
-
-func ApacheFeatureContext(s *godog.Suite) {
-	s.Step(`^Apache "([^"]*)" is running for metricbeat "([^"]*)"$`, apacheIsRunningForMetricbeat)
-	s.Step(`^metricbeat "([^"]*)" is installed and configured for Apache module$`, metricbeatIsInstalledAndConfiguredForApacheModule)
-	s.Step(`^there are no errors in the "([^"]*)" index$`, thereAreNoErrorsInTheIndex)
-}
```

Comment:
Great to see this generalized. If we need to add some specific step for a service, I guess that we can still create another context with its own steps? Or we'd need to add the steps to the general `FeatureContext`.

Reply:
This is something on my radar, so we must investigate it